Entity Explorer - IP Address.ipynb

```diff
@@ -104,49 +104,39 @@
    },
```

Comment:
<p>We shouldn't need WIDGET_DEFAULTS any more since this is created inside init_notebook but no harm leaving it there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>